src/server/services/updateMeters.js

```diff
@@ -24,7 +24,8 @@ async function updateAllMeters() {
 			metersToUpdate
 				.map(readMamacData)
 				.map(p => p.catch(err => {
-					log(`ERROR ON REQUEST TO ${err.options.uri}, ${err.message}`, 'error');
+					const uri = err.options !== undefined ? err.options.uri : '[NO URI AVAILABLE]';
```

Comment:
Please don't use the ternary operator like this. We can't stop John from doing it, but that doesn't mean the rest of us have to join in. It looks weird and also doesn't handle the case where `err.options` is defined but has no `uri` property.

I suggest:

```javascript
let uri = '[NO URI AVAILABLE]';
if (err.options !== undefined && err.options.uri !== undefined) {
    uri = err.options.uri;
}
```